Gruntfile.js

```diff
@@ -67,10 +67,11 @@ module.exports = function (grunt) {
       'softwerkskammer/build/stylesheets/less/agora.less'
     ]
   };
+  const integrationTestCoverageDataFolder =  'softwerkskammer/coverageIntegration';
   grunt.initConfig({
     clean: {
       build: ['softwerkskammer/build', 'softwerkskammer/frontendtests/fixtures/*.html'],
-      coverage: ['softwerkskammer/coverage', 'softwerkskammer/coverageWithDB', 'softwerkskammer/karma-coverage'],
+      coverage: ['softwerkskammer/coverage', integrationTestCoverageDataFolder, 'softwerkskammer/karma-coverage'],
```

Comment:
Hm ja, das ist verwirrend weil inkonsistent. Würde es helfen wenn wir die anderen Pfade in der Zeile ebenfalls in consts extrahieren?